app/src/main/java/io/github/droidkaigi/confsched2017/viewmodel/SessionsViewModel.java

```diff
@@ -56,6 +60,8 @@ public void destroy() {
                     this.rooms = extractRooms(sessions);
                     this.stimes = extractStimes(sessions);
 
+                    notifyPropertyChanged(BR.loadingVisibility);
```

Comment:
I don't have confidence, but I comment :)

If the project uses data binding, the visibility of views should be controlled by data binding as possible. However, if we use progress dialog instead of progress view in layout xml, it is difficult to control the visibility of dialog only with data binding and the method described in the article may be good.

In this case, I prefer progress bar than progress dialog because we use BottomNavigationView. Even if the session data is not loaded, users should be able to change tab.